src/USER-MISC/pair_agni.cpp

```diff
@@ -187,7 +188,13 @@ void PairAGNI::compute(int eflag, int vflag)
         const double wZ = cF*delz/r;
 
         for (k = 0; k < iparam.numeta; ++k) {
-          const double e = exp(-(iparam.eta[k]*rsq));
+          	double e = 0.0;
+
+		if(ver == 1) //FP1
+			e = exp(-(iparam.eta[k]*rsq));
+		else if(ver == 2) //FP2
+			e = (1 / (square(iparam.eta[k]) * iparam.gwidth * sqrt(2 * MathConst::MY_PI))) * exp(-(square(r - iparam.eta[k])) / (2 * square(iparam.gwidth)));
```

Comment:
- please use "1.0" instead of "1" for clarity when numbers are meant to be (and promoted to) floating point numbers.
- there is a `MathConst::MY_2PI`